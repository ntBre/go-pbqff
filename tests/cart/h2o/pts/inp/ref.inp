memory,1,g

nocompress;

geometry={
 3
 comment
 h          0.0000000000        0.7574590974        0.5217905143
 o          0.0000000000        0.0000000000       -0.0657441568
 h          0.0000000000       -0.7574590974        0.5217905143
}

basis={
default,cc-pVTZ-F12
}
set,charge=0
set,spin=0
hf
CCSD(T)-F12
