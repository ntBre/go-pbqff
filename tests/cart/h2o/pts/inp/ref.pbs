#!/bin/sh
#PBS -N H2O-ref
#PBS -S /bin/bash
#PBS -j oe
#PBS -o ref.inp.out
#PBS -W umask=022
#PBS -l walltime=5000:00:00
#PBS -l ncpus=1
#PBS -l mem=8gb

module load openpbs molpro

export WORKDIR=$PBS_O_WORKDIR
export TMPDIR=/tmp/$USER/$PBS_JOBID
cd $WORKDIR
mkdir -p $TMPDIR

date
hostname

molpro -t 1 ref.inp --no-xml-output
date

rm -rf $TMPDIR
