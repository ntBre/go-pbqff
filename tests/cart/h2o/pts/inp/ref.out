dummy output
energy=     -76.369839607972
GRADX(1:0)   = [ ] AU
GRADY(1:0)   = [ ] AU
GRADZ(1:0)   = [ ] AU
